incubator/dokuwiki/values.yaml

```diff
@@ -0,0 +1,63 @@
+## Bitnami dokuwiki image version
+## ref: https://hub.docker.com/r/bitnami/dokuwiki/tags/
+##
+image: bitnami/dokuwiki:20160626a-r0
+
+## Specify a imagePullPolicy
+## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+##
+# imagePullPolicy:
+
+## User of the application
+## ref: https://github.com/bitnami/bitnami-docker-dokuwiki#environment-variables
+##
+dokuwikiUsername: superuser
+
+## User's Full Name
+## ref: https://github.com/bitnami/bitnami-docker-dokuwiki#environment-variables
+##
+dokuwikiFullname: FullName
+
+## Application password
+## Defaults to a random 10-character alphanumeric string if not set
+## ref: https://github.com/bitnami/bitnami-docker-dokuwiki#environment-variables
+## dokuwikiPassword:  
+
+## Admin email
+## ref: https://github.com/bitnami/bitnami-docker-dokuwiki#environment-variables
+##
+dokuwikiEmail: user@example.com
+
+## 
+## ref: https://github.com/bitnami/bitnami-docker-dokuwiki#environment-variables
+##
+dokuwikiWikiname: Bitnami DokuWiki
```

Comment:
@prydonius should be keep this value when the chart is contributed to official? I would prefer `My Wiki`